### ISSUE TITLE
QR Code Bug

### ISSUE THREAD

**Issue by TheMiscreant** [helgoboss/realearn#431]
Hey there, you asked over on the Reaper forum to open an issue for the following (in your words): 

"It's not very convenient that way but it seems your particular network setup is a bit too complicated for ReaLearn to figure out the correct IP address automatically - it chooses the wrong one (maybe the one in use by your VPN adapter). If this is bothering you and you need the QR code convenience, please open a ticket on GitHub. Maybe I can improve the detection mechanism to prefer "192.168." (= local) IP addresses, or at least provide a possibility to enter a fixed IP address in ReaLearn's INI file."

**Comment by helgoboss** [comment]
Correcting myself: Fixed IP address probably doesn't make too much sense because most setups use DHCP. So preferring "192.168." is maybe the only way.

**Comment by TheMiscreant** [comment]
Yeah, honestly, not using the QR code isn't too big of a deal. I boot up, check my IP, boot up ReaLearn and input. Ta-da.

**Comment by helgoboss** [comment]
Not trivial because there's no code right now to enumerate all network adapters. The only thing we could do right now is logging a warning that the determined IP address doesn't look like a local address and therefore it probably won't work. Mmh.
